tests/acceptance/features/webUISharingExternal/federationSharing.feature

```diff
@@ -260,113 +130,83 @@ Feature: Federation Sharing - sharing with users on other cloud storages
     Then file "lorem (2).txt" should not be listed on the webUI
     And file "lorem (2).txt" should not be listed in the files page on the webUI
 
-  @skip @yetToImplement
-  Scenario: test sharing folder to a remote server and resharing it back to the local
```

Comment:
I think we need to have at least one test that tries to reshare federated share as federated share.

Reply:
Added.